src/language_negotiation_lambda.js

```diff
@@ -36,20 +35,26 @@ exports.handler = (event, context, callback) => {
     }
 
     try {
-        if (headers[acceptLanguage]) {
-            var selectableLanguages = getSelectableLangugaes(headers[acceptLanguage].split(','));
-            var selectedLanguage = defaultLanguage;
+        headers[customHeaderName] = [defaultLanguage];
+    } catch(err) {
+         console.log("Error assigning default language: " + err);
+    }
```

Comment:
This is just being super conservative about the format of the header object passed to us by cloudfront.  I wanted to make sure we caught it, but since we wrap `run()` in a try catch we can be assured that we do the right thing if this fails